src/Test.FunctionalTests.Sarif/TestData/Multitool/BaselineOption/ExpectedOutputs/TEST1001.ValidateWithBaseline.sarif

```diff
@@ -110,7 +110,6 @@
         {
           "ruleId": "SARIF2003",
           "ruleIndex": 0,
-          "level": "note",
```

Comment:
>"level": "note", [](http://example.com/codeflow?start=10&length=16)

these changes are the result of our fix in the SarifValidationSkimmerBase.cs file.

above we have 
"defaultConfiguration": {
    "level": "note"
},